spec/classes/server/config_spec.rb

```diff
@@ -14,7 +14,7 @@
       is_expected.to contain_package('policycoreutils-python').with(ensure: 'installed')
 
       is_expected.to contain_exec('/usr/sbin/semanage port -a -t postgresql_port_t -p tcp 5432')
-        .with(unless: '/usr/sbin/semanage port -l | grep -qw 5432')
+        .with(unless: ['/usr/sbin/semanage port -l | grep -qw 5432'])
```

Comment:
This was changed back to a string since it uses a shell anyway. Repeated in this file.

Reply:
fixed